src/spotted/handlers/warn.py

```diff
@@ -0,0 +1,36 @@
+"""/warn command"""
+from telegram import Update
+from telegram.ext import CallbackContext
+
+from spotted.data import Config, User
+from spotted.utils import EventInfo
+
+
+async def warn_cmd(update: Update, context: CallbackContext):
+    """Handles the /warn command.
+    Warn a user by replying to one of his message in the comment group with /warn
+
+    Args:
+        update: update event
+        context: context passed by the handler
+    """
+    info = EventInfo.from_message(update, context)
+    g_message = update.message.reply_to_message
+    if g_message is None:
+        await info.bot.send_message(
+            chat_id=info.chat_id,
+            text="Per warnare qualcuno, rispondi al suo commento con /warn",
+        )
+        return
+    user = User(g_message.from_user.id)
+    n_warns = user.get_n_warns()
+    text = f"L'utente {g_message.from_user.name} "
```

Comment:
We know that, technically, this feature can't be anonymous. But still, there is no need to show the subject's name in the admin group publicly. I would remove the identifier completely.
For example, the current /reply command says "L'utente ha ricevuto il messaggio", without indicating who the receiver is.